examples/powercli/eventbridge/eventbridge-secrets.json

```diff
@@ -0,0 +1,5 @@
+{
+    "AWS_REGION" : "us-west-2",
+    "AWS_ACCESS_KEY" : "XXXXXXXXXX",
+    "AWS_SECRET_KEY" : "XXXXXXXXXX"
+}
```

Comment:
See comment below to make EVENT_BUS configurable

Reply:
Fixed and pushed the changes already